tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py

```diff
@@ -304,16 +307,18 @@ def default_test(self, tables_and_test_values):
                 fields = [table_info.get('fields', [])[0]]
                 self.assertRowIDsMatch(fq_table_name, fields, values)
 
-            if self.query_class:
-                # test:  get the queries
-                query_list = self.query_class.get_query_specs()
-
+            if self.rule_instance:
                 # test: run the queries
-                engine.clean_dataset(self.project_id, query_list)
+                with mock.patch('cdr_cleaner.clean_cdr_engine.create_sandbox'
```

Comment:
This mock wouldn't be needed if sandbox_dataset_id were an argument of clean_dataset